src/promptflow/promptflow/_sdk/operations/_flow_operations.py

```diff
@@ -531,7 +531,7 @@ def build(
         :return: no return
         :rtype: None
         """
-        output_dir = Path(output)
+        output_dir = Path(output).absolute()
```

Comment:
why this fix the issue?

Reply:
because we will change working dir when export flow https://github.com/microsoft/promptflow/blob/43431fa4704e9a94c246c5ceed9d1ffe25b1ef50/src/promptflow/promptflow/_sdk/operations/_flow_operations.py#L364C1-L364C1

Not sure whether it's because we need generate connection inside flow folder before.